app/views/culinary_recipes/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="mx-auto md:w-2/3 w-full flex">
+  <%= form_with(model: @culinary_recipe, local: true) do |form| %>
+  <div class="flex items-center justify-center">
+    <%= form.check_box :public, class: "toggle-checkbox", id: "public-toggle" %>
+    <%= form.label :public, "Public", class: "toggle-label" %>
+  </div>
+<% end %>
+  <div class="mx-auto w-full">
+
+    <%= render @culinary_recipe %>
+
+    <%= link_to "Back to culinary_recipes", culinary_recipes_path, class: "ml-2 mt-10 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+
+    <%= link_to 'Generate Shopping List', method: :post, class: "l-2 mt-10 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+
```

Comment:
- You have done well so far, but your show page does not work as expected, when I click on the public recipe button, the details for the recipe does not get updated. Please fix this 🙏 
- Also, when I click on the Add new ingredient button, it should take me to a page where I can add the already added ingredients on the application, not to a form to create a new ingredient. Please fix this 🙏

Reply:
- The show page issue has been  resolved
<img width="1016" alt="Screenshot 2024-02-18 at 00 53 33" src="https://github.com/Anand8317/recipe/assets/58692590/8d3cf5ea-12ef-46c5-8741-a98952b760c1">